src/pybamm/models/full_battery_models/sodium_ion/basic_dfn.py

```diff
@@ -0,0 +1,273 @@
+#
+# Basic Doyle-Fuller-Newman (DFN) Model
+#
+import pybamm
+
+
+class BasicDFN(pybamm.lithium_ion.BaseModel):
+    """Doyle-Fuller-Newman (DFN) model of a sodium-ion battery, from
+    :footcite:t:`Marquis2019`.
```

Comment:
Does the Na-ion paper have the same model equations?  It would be good to reference that paper here for more context I think, and then say eqns are from Marquis (except they aren't quite ahah)

Reply:
Not quite, but because I took the parameters from the COMSOL implementation it is actually quite similar to the Marquis et al. Happy to cite either, not sure what will be less confusing to the users.